ocaml/xapi/extauth_plugin_ADwinbind.ml

```diff
@@ -31,6 +33,12 @@ let tdb_tool = !Xapi_globs.tdb_tool
 
 let debug_level = !Xapi_globs.winbind_debug_level |> string_of_int
 
+let call_wbinfo (args : string list) : (string, exn) result =
+  try
+    let stdout = Helpers.call_script ~log_output:Never wb_cmd args in
```

Comment:
if it is a generic function (other than only check krbtgt in retry_until_timeout), it should not set `log_outgput:Never` as we want the debug messages on exception.